### ISSUE TITLE
Membership report in a timeframe

### ISSUE THREAD

**Issue by goetzk** [tendenci/tendenci#616]
hi,
I'm wondering if it is possible to build a report (ideally accessible via reports app) that shows a breakdown of memberships per type over a certain range.

Is this doable at the moment or something that could be built in?

thanks,

**Comment by jennyq** [comment]
Have you looked at this one `/memberships/reports/members_ytd_type/`? I think it is very close to what you described. We can add some graphs to it.

**Comment by goetzk** [comment]
Um,
I thought I replied to this the next day, but it appears my email never made it here.

What I tried to convey at the time was: 

That is close to what I need, but I'm after a multi year record. On members_ytd_type I
can select year by year over four years but I am hoping for a single view
showing a chosen number of years. 

Graphs would, of course, be good for The Board but as long as the raw data can be generated it can be graphed in anything.

A combination of members_ytd_type and memberships/reports/members_over_time/ is what I envisage.
